acceptance/tests/subcommands/init.rb

```diff
@@ -4,24 +4,23 @@ def delete_root_folder_contents
     on default, 'rm -rf /root/*'
   end
 
-  step 'ensure that `beaker init` fails correctly when not provided a hypervisor' do
-    expect_failure('it should return a non-zero code when it fails') do
-      result = on(default, 'beaker init', :accept_all_exit_codes => true)
-      refute_equal(0, result.exit_code, '`beaker init` without a hypervisor argument should return a non-zero exit code')
+  step 'ensure that `beaker init` exit value should be 1 when not provided with a supported hypervisor' do
+    begin
+      on(default, 'beaker init ec2', :accept_all_exit_codes => true)
+    rescue SystemExit=>e
```

Comment:
It seems like this was used for the spec test, but the code here should just be saving the `result` object and then testing the `exit_code`. I would suspect that this `SystemExit` rescue is not actually rescuing anything at all.